### DISCUSSION TITLE
Vercel needs to improve login problem resolution flow

### DISCUSSION THREAD

**Discussion by rodrilima** [vercel/community#6080] [Account Issue]
### Question

I have used Vercel for a few years for some clients. I've already experienced several login problems and the platform itself doesn't provide alternatives to solving them other than to contact them via email and wait for someone to manually help. And that takes time.

I use other secure tools such as AWS, GCloud and I have never had as many problems logging in as with Vercel.

It's so annoying that I'm considering moving all my projects to another platform because of it. It is unacceptable that I cannot access my projects for any reason.

I believe that Vercel urgently needs to review login security flows, providing secure ways to authenticate when a problem occurs.

**Comment by leerob** [comment]
Hey @rodrilima, I'm sorry about the poor experience here. Could you share more details? Are you having issues logging into your same account, or is this about creating new accounts?

**Comment by rodrilima** [comment]
At this moment, I have a problem with my account. I even sent an email to support, but no response. I'm trying to log in and the page asks for my phone number. When I put it in it says that it is already linked to a Vercel account. And it's over. It doesn't matter if I try to log in with email or github. I can't log in because I have to enter my phone number and when I do, it says it's already linked to who knows which account.

**Reply by leerob** [reply]
Let me follow up with our support team and look into this for you. What is the support case number?

**Reply by rodrilima** [reply]
case number is 00206877

**Comment by amyegan** [comment]
Thanks for sharing those additional details and the case number @rodrilima. It could be a problem with the GitHub connection to your account. Do you run into the same problem if you [log in with email](https://vercel.com/login/email)?

**Reply by rodrilima** [reply]
Yes, same problem with email. It always asks for the phone number and when I put it in it says it is already linked to an account I don't know about.
